demo/vite.config.ts

```diff
@@ -3,7 +3,7 @@ import * as fs from 'fs'
 import url from 'url'
 import path from 'path'
 
-const cdnDomain = 'http://127.0.0.2:5173'
+const cdnDomain = 'http://127.0.0.1:5173'
```

Comment:
@CGNonofr Are you okay with this change? This was required on my side to make the icons/extensions work. I can imagine that maybe you have `127.0.0.1` as localhost?

Reply:
I can change it back easily if we want to merge this PR first.